### ISSUE TITLE
[accessibility] Fullscreen control should have `role="button"`

### ISSUE THREAD

**Issue by Malvoz** [Leaflet/Leaflet.fullscreen#110]
For consistency with the other controls:  
https://github.com/Leaflet/Leaflet/blob/bdd7ebf836201550f6d173943b728098b83f40d5/src/control/Control.Zoom.js#L89-L91

**Comment by Malvoz** [comment]
Other potentially helpful resources:
- [Leaflet Guideline: Test your maps](https://leafletjs.com/examples/accessibility/#test-your-maps)
- [Leaflet Plugin Authoring Guide: Content Accessibility](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md#content-accessibility)
